.circleci/config.yml

```diff
@@ -379,6 +356,8 @@ jobs:
       image: ubuntu-2204:2022.07.1
     steps:
       - checkout
+      - check-changed:
+          patterns: .*\.md$
```

Comment:
Would we want to use a check against the specs folder instead? This looks like it works, but IIRC the lints we use only apply to the specs folder (so we run just a tad more than intended).

Reply:
Yeah good point, will update